docs/earth-config/earth-config.md

```diff
@@ -37,23 +46,58 @@ git:
 
 Specifies the location where build data is stored. The default location is `/var/cache/earthly`.
 
+### cache_size_mb
+
+Specifies the total size of the buildkit cache, in MB. The buildkit daemon will allocate disk space for this size. Size less than `1000` (1GB) is not recommended. The default size if this option is not set is `10000` (10GB).
+
+This setting is only used when the cache is initialized for the first time. In order to apply the setting immediately, issue the following command after changing the configuration
+
+```bash
+earth prune --reset
```

Comment:
The closing back ticks are missing here?

Reply:
fixed, this was rather sloppy of me :(